packages/utilities/psammead-test-helpers/src/index.js

```diff
@@ -4,8 +4,13 @@ import deepClone from 'ramda/src/clone';
 
 export const shouldMatchSnapshot = (title, component) => {
   it(title, () => {
+    // Components with a single child do not have to be wrapped in a div when matching against a snapshot.
+    const removeWrappingDiv = container => container.firstChild;
     const { container } = render(component);
-    expect(container.firstChild).toMatchSnapshot();
+    const hasOneChild = container.children.length === 1;
```

Comment:
and then here 

`// if the container has more than one child then it's a component that uses a fragment at the top level so we should not select the first child because it wouldn't snapshot the whole component`

I'm in a rush so couldn't think of a more concise comment!

Reply:
I'll add it as a separate PR, but I don't think it needs to be any more concise as it it is very precise

Reply:
https://github.com/bbc/psammead/pull/1994

PR with the comment updates ^